src/projection/node/create-projection-node.ts

```diff
@@ -1677,3 +1699,18 @@ function roundBox(box: Box): void {
     roundAxis(box.x)
     roundAxis(box.y)
 }
+
+/**
+ * Look up the tree and check whether any element is performing a
+ * layout-affecting animation.
+ */
+function isTreeAnimatingLayoutAffectingStyle(
+    visualElement: VisualElement | undefined
+): boolean {
+    while (visualElement) {
+        if (Boolean(visualElement.layoutAffectingAnimations)) return true
+        visualElement = visualElement.parent
+    }
```

Comment:
What's the performance impact of this?
Would it help to memoize it?

Reply:
Actually I suppose because `warnOnce` is only going to fire once, this doesn't need to be checked a second time. So I can change it so the boolean passed to `warnOnce` is actually a resolver function that only gets run if we haven't already fired the warning

Reply:
Well, depends if we want to include the DOM element for which this happens, right?